code/datums/mutations/touch.dm

```diff
@@ -419,3 +419,4 @@
 	icon_state = "greyscale"
 	color = COLOR_VERY_PALE_LIME_GREEN
 	inhand_icon_state = "greyscale"
+	item_flags = ABSTRACT | HAND_ITEM
```

Comment:
```suggestion
	item_flags = parent_type::item_flags &= ~NEEDS_PERMIT
```
(Though imo it'd probably be cleaner to have the spell remove the flag in `create_hand`) (Maybe even as a var on `spell/touch`) (But doesn't matter ultimately)

Reply:
I'll have to have a look at it later. I think at the moment this only matters for this spell, but if we want to introduce more helpful touch spells, this probably could stand to have a var for it.